arviz/rcparams.py

```diff
@@ -93,11 +102,60 @@ def _validate_boolean(value):
     return value is True or value == "true"
 
 
+def make_iterable_validator(scalar_validator, length=None):
+    """Validate value is an iterable datatype."""
+    # based on matplotlib's _listify_validator function
+
+    def validate_iterable(value):
+        if isinstance(value, str):
+            val = tuple([scalar_validator(v.strip("([ ])")) for v in value.split(",") if v.strip()])
+            if length is not None:
+                if len(val) == length:
+                    return val
+                else:
+                    raise ValueError("Iterable must be of length: {}".format(length))
+            else:
+                return val
+        elif np.iterable(value) and not isinstance(value, (set, frozenset)):
+            val = tuple([scalar_validator(v) for v in value])
+            if length is not None:
+                if len(val) == length:
+                    return val
+                else:
+                    raise ValueError("Iterable must be of length: {}".format(length))
+            else:
+                return val
+        else:
+            raise ValueError("Only iterable values are valid")
+
+    return validate_iterable
+
+
+def _make_validate_float_iterable(length):
+    """Validate value is a float iterable of given length."""
+    # uses make_iterable_validator to validate float iterables
+
+    def validate_float_iterable(value):
+        if value is None or isinstance(value, str) and value.lower() == "none":
+            return None
+        elif value == "auto":
+            return value
+        try:
+            validate_floatlist = make_iterable_validator(_validate_float_or_none, length)
+            return validate_floatlist(value)
+        except:
+            raise ValueError("Only float iterable values are valid.")
+
+    return validate_float_iterable
+
+
 defaultParams = {  # pylint: disable=invalid-name
     "data.http_protocol": ("https", _make_validate_choice({"https", "http"})),
     "data.load": ("lazy", _make_validate_choice({"lazy", "eager"})),
     "data.index_origin": (0, _make_validate_choice({0, 1}, typeof=int)),
     "plot.backend": ("matplotlib", _make_validate_choice({"matplotlib", "bokeh"})),
+    "plot.bokeh.bounds_x_range": ("auto", _make_validate_float_iterable(length=2)),
+    "plot.bokeh.bounds_y_range": ("auto", _make_validate_float_iterable(length=2)),
```

Comment:
And here use the `_validate_bokeh_bounds`

Reply:
Yes this is way more optimized! I'll definitely make the changes.

Also, how can I add a ```backend_config``` option to enable/disable these limits as suggested in #959 by @ahartikainen. Is it like an argument that the user can pass to ```plot_autocorr```?

Reply:
Yes, it would be a new argument to plot_autocorr so that the user can pass to it a dictionary with keys like `bounds_x_range`. The preference would then be to use the value in `backend_config` if present, otherwise use the value in rcParams

Reply:
Isn't it better to include it in ```backend_kwargs``` so that other plots can also use it?

Reply:
`backend_kwargs` is currently used only at axes/figure creation (some PRs ago we decided on not including `show` there because show is not relevant on creation but after plotting). I think this is similar to `show` case? Or can this be defined at creation? (I'm still getting up to date with bokeh)

Also, do not worry too much about other plots for now. Adding this in `backend_kwargs` will mean having to modify their code and use this new key whereas using `backend_config` is adding a new argument and using it.

Reply:
I don't have much knowledge either but for simple plots (single plot) I've seen that these bounds are passed at the time of figure creation only. But for gridplots I don't think these are needed at the time of axes creation and can be set separately (as done here).

By other plots I meant only those plots for which these bounds have been added:
- autocorrplot
- traceplot
- parallelplot
- forestplot
I should add the ```backend_config``` to these plots only for now right?